woocommerce/woocommerce-android
# Issue/3726 capture payment endpoint

Parent issue #3726

Merge instructions:
1. Make sure FluxC's PR is merged first - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1980
2. Merge this PR

This PR 
- Updates FluxC hash.
- Updates "capturePayment" method so it uses WCPayStore.capturePayment.
- Propagates "orderId" to "capturePayment".

To test:
Note: Since the backend code is not in production yet, this PR needs to be tested against a development version of WCPay plugin. There are four options:
1. Build https://github.com/Automattic/woocommerce-payments/pull/1689 locally and upload the plugin to your testing site
2. DM me and I can send you the zip with the build
3. DM me and I can invite you to my testing site
4. Ask iOS folks to test this PR since they have the setup ready

Instructions:
- Make sure to test these changes with a site with WCPay in devmode (or live mode, but only hw reader is supported in live mode) and with development version of WCPay as mentioned above
- HW reader is required to test this PR
1. Open app settings
2. Tap on "Manage card reader"
4. Tap on "Connect card reader"
5. Go back - open a detail of an order in **USD currency**
6. Tap on "Collect payment"
7. Check that the flow works and the payment succeeds
8. Dismiss the dialog -> pull to refresh the order detail -> notice the order is "Paid"

Update release notes:
